changelogs/6.6.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[release-notes-6.6]]
+== APM Server version 6.6
+
+https://github.com/elastic/apm-server/compare/6.5\...6.6[View commits]
+
+* <<release-notes-6.6.0>>
+
+[[release-notes-6.6.0]]
+=== APM Server version 6.6.0
+
+https://github.com/elastic/apm-server/compare/v6.5.4\...v6.6.0[View commits]
+
+[float]
+==== Added
+
+- Add `span.sync` property to intake json spec and index field in ES. {pull}1548[1548].
+- Make `service.framework` properties optional and nullable {pull}1546[1546].
+- Add optional `span.subtype` and `span.action` properties. {pull}1618[1618].
+- Add `transaction.sampled` to errors {pull}1662[1662].
```

Comment:
@roncohen was there no changelog entry on purpose for https://github.com/elastic/apm-server/pull/1630/files? I am missing that here for 6.6.